lib/Conversion/TorchToLinalg/Uncategorized.cpp

```diff
@@ -2648,4 +3035,7 @@ void mlir::torch::torch_to_linalg::populateUncategorizedPatternsAndLegality(
   patterns.add<ConvertDequantizePerChannel>(typeConverter, context);
   target.addIllegalOp<AtenGridSamplerOp>();
   patterns.add<ConvertAtenGridSamplerOp>(typeConverter, context);
+  target.addIllegalOp<Aten__InterpolateSizeListScaleListOp>();
+  patterns.add<ConvertBilinearInterpolateOp>(typeConverter, context);
+  patterns.add<ConvertInterpolateNearestOp>(typeConverter, context);
```

Comment:
@aldesilv, I think a target is missing here.

Reply:
one is for the nearest mode the other is for bilinear